cellmlmanip/data/cellml_1_0.rng

```diff
@@ -0,0 +1,989 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
Should we include the .rnc source for this too, for completeness?

Reply:
Just in case someone wants to extract it?

Reply:
Sure!

Reply:
I've also stuck them here, by the way: https://github.com/MichaelClerx/cellml-validation/tree/master/cellml_1_0
Hope you don't mind!
([it works well but gets beaten by OpenCOR](https://github.com/MichaelClerx/cellml-validation/tree/master/reports))

Reply:
Nice :)